documentation/developer-guide/modules/tooling-guide/pages/java-aspect-tooling.adoc

```diff
@@ -439,3 +439,82 @@ characteristicDefinitionsUrl.ifPresent( url -> {
+the concept of Submodels, which describe certain aspects of a Digital Twin.
+
+The BAMM Aspect Meta Model allows to specify aspects of a digital twin and its semantics.
+The AAS Generator module provides mapping implementations to derive AAS Submodels from BAMM Aspect models
+and by that allows to on the one hand integrate BAMM models in AAS environments and on the other hand allow
+AAS Submodels to be described with rich semantics, as it is possible with BAMM.
+
+[[details-mapping-aas]]
+== Details to the Mapping Concept
+In the following section, the mapping rules applied by the generator are explained.
+The rules apply to https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/index.html[BAMM v1.0.0] and
+https://www.plattform-i40.de/IP/Redaktion/EN/Downloads/Publikation/Details_of_the_Asset_Administration_Shell_Part1_V3.html[AAS Specification Part 1 V3.0RC01].
+
+[cols="1,1,2"]
+|===
+| BAMM  | AAS  | Comment
+
+| **bamm:Aspect**  |  aas:Submodel with kind=Template  | Empty Asset and AssetAdministrationShell entries are added to the output file
+
+| bamm:name   | aas:Submodel.idShort  |
+
+| bamm:preferredName   | aas:Submodel.displayName  |
+
+| bamm:description   | aas:Submodel.description  |
+
+| bamm:property   | see **bamm:Property**  |
+
+| bamm:operation   | see **bamm:Operation**   |
+
+| bamm:Aspect.urn  | aas:Submodel.semanticId  |
+
+| **Property**  | aas:Property <br> aas:SubmodelElementCollection   | The AAS type is derived from the type of the BAMM Characteristic specifying the BAMM property. Depending on the type it is decided what the resulting AAS element will be. In case of an Entity it will result in a SubmodelElementCollection. It will also be a SubmodelElementCollection if the BAMM Characteristic is of a Collection type (see the https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/characteristics.html[Characteristics taxonomy]). In all other cases an aas:Property will be generated
+
+| bamm:Property.name  | aas:Property.idShort |
+
+| bamm:Property.urn  | aas:ConceptDescription.identification.id <br> aas:Property.semanticId|
+
+| bamm:Property.preferredName   | aas:Property.displayName  |
+
+| bamm:Property.description   | aas:Property.description  |  Note: Also mapped to aas:DataSpecificationIEC61360.definition of the aas:ConceptDescription of this property
+
+| bamm:Property.exampleValue  |aas:Property.value  |
+
+| bamm:Characteristic.dataType  | aas:Property.valueType |
+
+| **Operation** |  Operation  | in/out parameters are not used in BAMM so the mapping only generates input variables and output variables in AAS
```

Comment:
Why not "bamm:Operation"? Question also applies to several cells below in this column.